packages/darklang/languageTools/writtenTypes.dark

```diff
@@ -50,6 +50,7 @@ module Darklang =
           | TFloat of SourceRange
           | TChar of SourceRange
           | TString of SourceRange
+          | TList of SourceRange * TypeReference.TypeReference
```

Comment:
I think this is missing:
- the SourceRange of the `List` word itself
- the SourceRange of `<` and of `>`

maybe it's overkill to include those? I think including them would be consistent with our def of `EInt8` for example, though, where we include SourceRanges for symbols